properties/base.py

```diff
@@ -278,14 +281,20 @@ def assert_valid(self, instance, value=None):
             value.validate()
         return True
 
-    @staticmethod
-    def as_json(value):
+    def as_json(self, value):
```

Comment:
currently there is a pickle and a serialize call that are different but really doing the same thing.